packages/smooth_app/lib/background/abstract_background_task.dart

```diff
@@ -67,7 +59,10 @@ abstract class AbstractBackgroundTask {
   User getUser() => User.fromJson(jsonDecode(user) as Map<String, dynamic>);
 
   /// Downloads the whole product, updates locally.
-  Future<void> _downloadAndRefresh(final LocalDatabase localDatabase) async {
+  ///
+  /// Returns true if successful.
```

Comment:
You should add some brackets around the true (small change, I know ^^).

Reply:
Actually I should _not_ according to https://dart.dev/guides/language/effective-dart/documentation:
```dart
/// True if this box contains a value.
bool get hasValue => _value != null;

/// Returns `true` if every element satisfies the [predicate].
bool all(bool predicate(T element)) => ...
```

DO use square brackets in doc comments to refer to in-scope identifiers (... variable, method, or type names ...)
https://dart.dev/guides/language/effective-dart/documentation#do-use-square-brackets-in-doc-comments-to-refer-to-in-scope-identifiers